README.md

```diff
@@ -1,7 +1,7 @@
 # Socket collection
 
-|Crate|Documentation|Linux/OS X|Windows|
-|:---:|:-----------:|:--------:|-------|
-| [![](http://meritbadge.herokuapp.com/socket-collection)](https://crates.io/crates/socket-collection) | [![Documentation](https://docs.rs/socket-collection/badge.svg)](https://docs.rs/socket-collection) | [![Build Status](https://travis-ci.org/ustulation/socket-collection.svg?branch=master)](https://travis-ci.org/ustulation/socket-collection) | [![Build status](https://ci.appveyor.com/api/projects/status/ajw6ab26p86jdac4/branch/master?svg=true)](https://ci.appveyor.com/project/MaidSafe-QA/socket-collection/branch/master)
+|Crate|Documentation|Linux/OS X/Windows|
+|:---:|:-----------:|:-----------------|
```

Comment:
i think if you don't put the ending `:` it'll left justify the stuff in the table - we want centre justification so have it on both sides for the last column too. Try to view the file ("View file" above) in github to get an idea of how it will look once it is rendered